pkg-manager/core/src/install/extendInstallOptions.ts

```diff
@@ -187,7 +188,7 @@ const defaults = (opts: InstallOptions) => {
     },
     lockfileDir: opts.lockfileDir ?? opts.dir ?? process.cwd(),
     lockfileOnly: false,
-    nodeVersion: process.version,
+    nodeVersion: opts.nodeVersion ?? getSystemNodeVersion(),
```

Comment:
This should be enough
```suggestion
    nodeVersion: opts.nodeVersion,
```

Reply:
The issue is that we pass `process.version`. `getSystemNodeVersion` will be used anyway inside `package-is-installable`.